pages/common/xmlstarlet.md

```diff
@@ -0,0 +1,33 @@
+# xmlstarlet
+
+> A commandline XML/XSLT toolkit.
+> Note: You will likely need to know XPath: <https://developer.mozilla.org/en-US/docs/Web/XPath>.
+> More information: <https://xmlstar.sourceforge.net/docs.php>.
+
+- Format an XML document and print to stdout:
+
+`xmlstarlet format {{path/to/file.xml}}`
+
+- XML document can also be piped from stdin:
+
+`{{cat path/to/file.xml}} | xmlstarlet format`
+
+- Print all nodes that match a given XPath:
+
+`xmlstarlet select --template --copy-of {{xpath}} {{file.xml}}`
+
+- Insert an attribute to all matching nodes, and print to stdout (source file is unchanged):
+
+`xmlstarlet edit --insert {{xpath}} --type attr --name {{attribute_name}} --value {{attribute_value}} {{file.xml}}`
+
+- Rename some nodes, delete some nodes, and print to stdout (source file is unchanged):
+
+`xmlstarlet edit --rename {{xpath_to_rename}} --value {{new_name}} --delete {{xpath_to_delete}} {{file.xml}}`
+
+- Escape or unescape special XML characters in a given string:
+
+`xmlstarlet [un]escape {{string}}`
```

Comment:
```suggestion
`xmlstarlet {{escape|unescape}} {{string}}`
```

Reply:
I don't think this is the correct usage of `{{a|b}}` syntax according to style-guide.md.

`escape` and `unescape` here are literals, not placeholders.